aodntools/timeseries_products/Documentation/aggregated_timeseries.md

```diff
@@ -0,0 +1,175 @@
+# Non-Velocity Aggregated Product
+
+- [Objective](#objective)
+- [Scope](#scope)
+- [Input](#input)
+- [Output](#output)
+- [Method](#method)
+- [Distribution](#distribution)
+
+
+
+## Objective
+
+This product project will provide:
+
+A comprehensive set of aggregated time-series files for each mooring site and parameter (excluding current velocity), 
+without any interpolation or filtering, except for the exclusion of the out-of-water data. The Quality Control (QC) 
+flags will be preserved. All the (python) code used for the generation of the products openly available on GitHub.
+Clear documentation on the code and the output file format.
+
+## Scope
+
+Input Files and variables to be aggregated by this project will meet the following requirements:
+
+- File contains data from only one deployment of one instrument;
+- File is a delayed-mode, quality-controlled product (file version label “FV01”);
+- File is compliant with CF-1.6 and IMOS-1.4 conventions;
+- File contains, at the minimum, the variable of interest (VoI), and variables TIME, LATITUDE, LONGITUDE, and either variable NOMINAL_DEPTH or global attribute instrument_nominal_depth;
+- All files to be aggregated are from the same site, and have the same site_code attribute;
+- Variables to be aggregated have TIME as their only dimension (or if LATITUDE and LONGITUDE are included as dimensions, they have size 1);
+- Variables to be aggregated do not represent current velocity (these will be included in another product); and
+- The in-water data are bounded by the global attributes time_deployment_start and time_deployment_end.
+
+## Input
+
+The aggregation function will accept a list of input files, and the name of the variable of interest (VoI).
+
+The code will be able to aggregate variables and files that meet the following requirements:
+
+- File contains data from only one deployment of one instrument;
+- File is a delayed-mode product, with file version label FV01;
+- File is compliant with CF-1.6 and IMOS-1.4 conventions;
+- File contains, at the minimum, the VoI, and variables TIME, LATITUDE, LONGITUDE, and   NOMINAL_DEPTH;
```

Comment:
It looks like you copied this from an earlier version of the IP document. The latest version says 
* File contains, at the minimum, the VoI, and variables TIME, LATITUDE, LONGITUDE, and either variable NOMINAL_DEPTH or global attribute instrument_nominal_depth;

I think the rest of the dot points are unchanged.

Reply:
fixed